package.json

```diff
@@ -134,6 +135,7 @@
     "karma-jasmine": "^1.1.0",
     "karma-mocha-reporter": "^2.2.3",
     "karma-remap-coverage": "^0.1.4",
+    "karma-sonarqube-unit-reporter": "0.0.13",
```

Comment:
can you move this to the readme so it's not downloaded then check if it's installed when you run karma

Reply:
Done.. and updated the Readme .. Do let me know if this is what you meant